lib/web/desktop_playback.go

```diff
@@ -49,16 +50,6 @@ func (h *Handler) desktopPlaybackHandle(
 		return nil, trace.Wrap(err)
 	}
 
-	upgrader := websocket.Upgrader{
-		ReadBufferSize:  4096,
```

Comment:
Note that we will now be using buffers that are 75% smaller. Does this make sense for desktop recordings where we have larger messages?

Reply:
My sense is that it can make a difference on the margins, but we'd need to do some sort of profiling to know the true impact.